### PR TITLE
Deprecate db and resolvers generator

### PR DESCRIPTION
**Pull request by wtrocki**
This PR started as simple change but I have been continuing work by picking nondestructive and conflicting changes 

1) Remove cmd DB migrations (mostly docs and cmd - all removals so get)
We need to do this to have a unified way to solve migrations and address numerous issues from the community: #1095 etc.

2) Remove graphql-codegen-resolvers integration from source code, while keeping resolvers plugin still alive for a while to not trigger conflicts with other PR's etc.

3) Cleanup templates and give the ability to compose them together.
We have added the ability to clean up templates so we can use multiple layers of them when doing init. This enabled us to have multiple variations runtime templates with little effort. I still kept the 

4) Remove migration config and improvements 

The purpose of this PR is to bring the changes that will minimize the amount of the work required for the CRUD services while still being mostly deleting and reorganizing things. 

## Verification

1. cd packages/graphback-cli
2. yarn build
3. yarn cmdinit 
4. Pick first template (it composes from 2 templates)
5. All should be able to compose 
6. Template should be runnable

### CODE REVIEW

File: docs/releases.md [hunk 1]
```diff
@@ -10,11 +10,18 @@ Please follow individual releases for more information.
 
 # 0.14.0
```
Comment (machi1990) [inline docs/releases.md:11]: :-)

**Comment by craicoverflow** [comment]
These are being removed not deprecated

File: .vscode/launch.json [hunk 2]
```diff
@@ -28,37 +28,11 @@
         "DB_HOST": "localhost",
         "DB_PORT": "55432"
       },
-      "args": [
```
Comment (craicoverflow) [inline .vscode/launch.json:31]: You still need to have an "args" config to point to the file to run the app.

**Comment by wtrocki** [comment]
Resolver plugin is actually staying there it is simply deprecated by not mentioning it in the future docs. Previous docs will still get it and we going to release new version of the resolver plugin to people despite not having any templates there. 

As for CLI I was looking for an effective way to deprecate command and I found that deprecating it will make no impact. If we remove codegen template there will be no way to use it anyway so having it here - even as deprecated could cause potential harm (especially that some of the migration stuff will need to be refactored to cather new CRUD). 

I think we reached the point where we need to 
I'm going to create new document to show how things could play together and draft the architecture of the datasync in the diagram that we all can relate too.

File: .vscode/launch.json (hunk 2 above)
Comment (craicoverflow) [reply .vscode/launch.json:31]: Add this:

```
"args": [
   "${workspaceRoot}/templates/ts-apollo-postgres-backend/src/index.ts"
],
```

**Comment by wtrocki** [comment]
@craicoverflow I will help with rebasing your changes. Fixing tests

**Pull request merged**